<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QWidget" name="UserForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/images/nodeOpr.png</normaloff>:/img/images/nodeOpr.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>461</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>用户管理</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>用户列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修 改</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_type">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>操作</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>超级</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_password">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_user">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删 除</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>类型</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_insert">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添 加</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../img.qrc"/>
 </resources>
 <connections/>
</ui>
